rpc/handlers.go

```diff
@@ -1031,9 +1031,15 @@ func (h *Handler) TransactionStatus(ctx context.Context, hash felt.Felt) (*Trans
 	case ErrTxnHashNotFound:
 		txStatus, err := h.feederClient.Transaction(ctx, &hash)
 		if err != nil {
+			// Handle other internal errors
```

Comment:
Let's remove this comment. I don't think it's relevant any more.

Reply:
done